### PR TITLE
Added docs about CF_REMOTE_SSH_KEY env variable

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: contrib/cf-remote/README.md [hunk 1]
```diff
@@ -85,6 +85,14 @@ Note that this demo setup (`--demo`) is notoriously insecure.
 It has default passwords and open access controls.
 Don't use it in a production environment.
 
+### Specify an SSH key
+
+If you have more than one key in `~/.ssh` you may need to specify which key `cf-remote` is to use.
+
+```
+$ export CFREMOTE_SSH_KEY="~/.ssh/id_rsa.pub"
```
Comment (olehermanse) [inline contrib/cf-remote/README.md]: `CF_REMOTE_SSH_KEY`*

Comment (craigcomstock) [reply contrib/cf-remote/README.md]: will pull, squash and push this change tomorrow.

**Comment by nickanderson** [comment]
Squash and merge?

**Comment by nickanderson** [comment]
Nitpick on Commit message, use past tense.

**Pull request merged**